src/bubo-cache.cc

```diff
@@ -23,8 +23,8 @@ BuboCache::~BuboCache() {
     delete strings_table_;
 }
 
-void BuboCache::initialize(v8::Local<v8::Object> ignoredAttrs) {
-    ignored_attributes_.Reset(ignoredAttrs);
+void BuboCache::initialize(std::vector<std::string> ignoredAttrs) {
```

Comment:
I always do whatever has the least ampersands, but that's just because I'm a bewildered chicken. I'll put some ampersands in.